### ISSUE TITLE
Incomplete output from the converter

### ISSUE THREAD

**Issue by cstpierr** [AquaticInformatics/flowtracker2-field-data-plugin#32]
The .dis file generated from the converted do not include the leading zero for the single digit measurement points (i.e. below 10) leading to improper ingestion by Aquarius.

**Comment by DougSchmidt-AI** [comment]
Thanks for reporting the problem.

I can't reproduce the problem here using the converter with any of my FlowTracker2 files. The converted `.DIS` files are importing just fine for me.

Can you attach both the original FlowTracker2 file and the converted DIS file so I can take a deeper look?

GitHub Issues don't support `*.ft` or `*.dis` file extensions by default, so you'll need to rename the files with one of the supported file extensions. You can just append a ".zip" to the FT2 filename, and a ".txt" to the converted DIS filename.

```
OriginalMeasurement.ft  => OriginalMeasurement.ft.zip
OriginalMeasurement.dis => OriginalMeasurement.dis.txt
```

Then you should be able to drag and drop those renamed files into the comment box below and I'll be able to take a look.

**Comment by DougSchmidt-AI** [comment]
Ah, I saw sample files your colleague reported via our Support Portal.

```
St  Clock     Loc  Depth   IceD %Dep  MeasD Npts Spike     Vel   SNR Angle    Verr Bnd    Temp CorrFact   MeanV   Area     Flow   %Q
()     ()     (m)    (m)    (m) (*D)    (m)   ()    ()   (m/s)  (dB) (deg)   (m/s)  ()  (degC)       ()   (m/s)  (m^2)  (m^3/s)  (%)
 0  10:45    0.50  0.000  0.000  0.0  0.000    0     0   0.000   0.0     0  0.0000   0    0.00     1.00  0.5900  0.000   0.0000  0.0
 1  10:46    0.83  0.080  0.000  0.6  0.080   80     0   0.590  35.5    -6  0.0141   0   13.17     1.00  0.5900  0.026   0.0153  3.8
 2  10:52    1.15  0.100  0.000  0.6  0.100   80     3   0.732  38.0    -3  0.0108   0   13.21     1.00  0.7324  0.033   0.0238  5.8
 3  10:57    1.48  0.140  0.000  0.6  0.140   80     1   0.424  35.9     2  0.0166   0   13.24     1.00  0.4239  0.046   0.0193  4.7
 4  11:01    1.80  0.160  0.000  0.6  0.160   80     1   0.525  34.7   -17  0.0153   0   13.25     1.00  0.5251  0.052   0.0273  6.7
 5  11:04    2.13  0.170  0.000  0.6  0.170   80     1   0.910  34.5    10  0.0206   0   13.28     1.00  0.9096  0.055   0.0503 12.3
 6  11:07    2.45  0.160  0.000  0.6  0.160   80     0  -0.094  34.7  -151  0.0136   0   13.32     1.00 -0.0941  0.052  -0.0049 -1.2
 7  11:12    2.78  0.170  0.000  0.6  0.170   80     1   0.502  34.4    13  0.0243   0   13.34     1.00  0.5015  0.055   0.0277  6.8
 8  11:14    3.10  0.200  0.000  0.6  0.200   80     1   0.526  39.4    -1  0.0127   0   13.35     1.00  0.5256  0.065   0.0342  8.4
 9  11:16    3.43  0.160  0.000  0.6  0.160   80     1   0.329  42.3   -16  0.0192   0   13.39     1.00  0.3288  0.052   0.0171  4.2
10  11:21    3.75  0.140  0.000  0.6  0.140   80     0   0.630  36.7   -10  0.0130   0   13.43     1.00  0.6298  0.046   0.0287  7.0
11  11:25    4.08  0.120  0.000  0.6  0.120   80     0   0.565  35.0    -7  0.0129   0   13.46     1.00  0.5645  0.039   0.0220  5.4
12  11:26    4.40  0.100  0.000  0.6  0.100   80     0   0.705  35.6     4  0.0100   0   13.46     1.00  0.7045  0.033   0.0229  5.6
13  11:28    4.73  0.120  0.000  0.6  0.120   80     0   0.409  37.1    -1  0.0154   0   13.50     1.00  0.4088  0.039   0.0159  3.9
14  11:30    5.05  0.080  0.000  0.6  0.080   80     1   0.768  40.0    -1  0.0114   0   13.54     1.00  0.7683  0.026   0.0200  4.9
15  11:33    5.38  0.140  0.000  0.6  0.140   80     4   0.109  39.6    -4  0.0129   0   13.55     1.00  0.1090  0.046   0.0050  1.2
16  11:36    5.70  0.140  0.000  0.6  0.140   80     0   0.513  41.5   -16  0.0130   0   13.57     1.00  0.5126  0.046   0.0233  5.7
17  11:38    6.03  0.130  0.000  0.6  0.130   80     0   0.578  42.2     0  0.0130   0   13.61     1.00  0.5776  0.042   0.0244  6.0
18  11:41    6.35  0.130  0.000  0.6  0.130   80     2   0.704  39.2    -4  0.0070   0   13.65     1.00  0.7036  0.042   0.0297  7.3
19  11:43    6.68  0.160  0.000  0.6  0.160   80     0   0.113  16.1    -2  0.0062   0   13.68     1.00  0.1134  0.052   0.0059  1.4
20  11:49    7.00  0.000  0.000  0.0  0.000    0     0   0.000   0.0     0  0.0000   0    0.00     1.00  0.1134  0.000   0.0000  0.0
```

That brings in this crazy panel overview:
![image](https://user-images.githubusercontent.com/54773/55659859-5600c200-57b8-11e9-99f5-aaa8a8b5bcfc.png)

I can reproduce it with that data file, so I'll take a look.

**Comment by DougSchmidt-AI** [comment]
Fixed by #33 in [v17.4.42](https://github.com/AquaticInformatics/flowtracker2-field-data-plugin/releases/download/v17.4.42/FlowTracker2Converter.exe) of the `FlowTracker2Converter.exe`

```
St  Clock     Loc  Depth   IceD %Dep  MeasD Npts Spike     Vel   SNR Angle    Verr Bnd    Temp CorrFact   MeanV   Area     Flow   %Q
()     ()     (m)    (m)    (m) (*D)    (m)   ()    ()   (m/s)  (dB) (deg)   (m/s)  ()  (degC)       ()   (m/s)  (m^2)  (m^3/s)  (%)
00  10:45    0.50  0.000  0.000  0.0  0.000    0     0   0.000   0.0     0  0.0000   0    0.00     1.00  0.5900  0.000   0.0000  0.0
01  10:46    0.83  0.080  0.000  0.6  0.080   80     0   0.590  35.5    -6  0.0141   0   13.17     1.00  0.5900  0.026   0.0153  3.8
02  10:52    1.15  0.100  0.000  0.6  0.100   80     3   0.732  38.0    -3  0.0108   0   13.21     1.00  0.7324  0.033   0.0238  5.8
03  10:57    1.48  0.140  0.000  0.6  0.140   80     1   0.424  35.9     2  0.0166   0   13.24     1.00  0.4239  0.046   0.0193  4.7
04  11:01    1.80  0.160  0.000  0.6  0.160   80     1   0.525  34.7   -17  0.0153   0   13.25     1.00  0.5251  0.052   0.0273  6.7
05  11:04    2.13  0.170  0.000  0.6  0.170   80     1   0.910  34.5    10  0.0206   0   13.28     1.00  0.9096  0.055   0.0503 12.3
06  11:07    2.45  0.160  0.000  0.6  0.160   80     0  -0.094  34.7  -151  0.0136   0   13.32     1.00 -0.0941  0.052  -0.0049 -1.2
07  11:12    2.78  0.170  0.000  0.6  0.170   80     1   0.502  34.4    13  0.0243   0   13.34     1.00  0.5015  0.055   0.0277  6.8
08  11:14    3.10  0.200  0.000  0.6  0.200   80     1   0.526  39.4    -1  0.0127   0   13.35     1.00  0.5256  0.065   0.0342  8.4
09  11:16    3.43  0.160  0.000  0.6  0.160   80     1   0.329  42.3   -16  0.0192   0   13.39     1.00  0.3288  0.052   0.0171  4.2
10  11:21    3.75  0.140  0.000  0.6  0.140   80     0   0.630  36.7   -10  0.0130   0   13.43     1.00  0.6298  0.046   0.0287  7.0
11  11:25    4.08  0.120  0.000  0.6  0.120   80     0   0.565  35.0    -7  0.0129   0   13.46     1.00  0.5645  0.039   0.0220  5.4
12  11:26    4.40  0.100  0.000  0.6  0.100   80     0   0.705  35.6     4  0.0100   0   13.46     1.00  0.7045  0.033   0.0229  5.6
13  11:28    4.73  0.120  0.000  0.6  0.120   80     0   0.409  37.1    -1  0.0154   0   13.50     1.00  0.4088  0.039   0.0159  3.9
14  11:30    5.05  0.080  0.000  0.6  0.080   80     1   0.768  40.0    -1  0.0114   0   13.54     1.00  0.7683  0.026   0.0200  4.9
15  11:33    5.38  0.140  0.000  0.6  0.140   80     4   0.109  39.6    -4  0.0129   0   13.55     1.00  0.1090  0.046   0.0050  1.2
16  11:36    5.70  0.140  0.000  0.6  0.140   80     0   0.513  41.5   -16  0.0130   0   13.57     1.00  0.5126  0.046   0.0233  5.7
17  11:38    6.03  0.130  0.000  0.6  0.130   80     0   0.578  42.2     0  0.0130   0   13.61     1.00  0.5776  0.042   0.0244  6.0
18  11:41    6.35  0.130  0.000  0.6  0.130   80     2   0.704  39.2    -4  0.0070   0   13.65     1.00  0.7036  0.042   0.0297  7.3
19  11:43    6.68  0.160  0.000  0.6  0.160   80     0   0.113  16.1    -2  0.0062   0   13.68     1.00  0.1134  0.052   0.0059  1.4
20  11:49    7.00  0.000  0.000  0.0  0.000    0     0   0.000   0.0     0  0.0000   0    0.00     1.00  0.1134  0.000   0.0000  0.0
```

![image](https://user-images.githubusercontent.com/54773/55660654-5b600b80-57bc-11e9-8400-a6068736f54f.png)
